sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/ConfluentSchemaRegistryDeserializerProvider.java

```diff
@@ -60,7 +60,8 @@
       SerializableFunction<Void, SchemaRegistryClient> schemaRegistryClientProviderFn,
       String schemaRegistryUrl,
       String subject,
-      @Nullable Integer version) {
+      @Nullable Integer version,
+      @Nullable Map<String, ?> schemaRegistryConfigs) {
```

Comment:
Do we really need to pass this argument in constructor? I don't see that is used after.

Reply:
@aromanenko-dev I've removed this augment from the constructor, PTAL. thanks!